### DISCUSSION TITLE
Welcome to mikbot Discussions!

### DISCUSSION THREAD

**Discussion by DRSchlaubi** [DRSchlaubi/mikbot#155] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by louboecker** [comment]
Hi

**Comment by alexrandaccio** [comment]
Hey there! I am trying to stand up an instance of VoteBot for testing. I cloned the repo and followed the instructions in the README.

Could I have some advice on how to begin testing it? I've never developed a Discord bot before

**Reply by louboecker** [reply]
You don't have to clone the repository to use mikbot, if you have a working docker environment you can take a look at [this installation guide](https://github.com/DRSchlaubi/mikbot/blob/main/SETUP.md) and add the `votebot` plugin like explained in [this section](https://github.com/DRSchlaubi/mikbot/blob/main/SETUP.md#add-any-plugin) there's also an explanation for the environment variables in the [Readme of VoteBot](https://github.com/DRSchlaubi/mikbot/tree/main/votebot#self-hosting)
